ang/civiawards/reviews-tab/directives/reviews-case-tab-content.html

```diff
@@ -41,14 +41,14 @@
               dropdown-append-to="'#bootstrap-theme'"
             >
               <button
-                class="btn btn-default btn-sm dropdown-toggle"
+                class="btn btn-default btn-sm dropdown-toggle civiawards__reviews-tab__record-action-button"
```

Comment:
@reneolivo can't we make a generic button for above a panel with a shadow etc? That would be standard.